src/screens/lifemap/FamilyMembersNames.js

```diff
@@ -125,16 +135,52 @@ export class FamilyMembersNames extends Component {
     })
   }
 
+  getFamilyMembersCountArray = t => [
```

Comment:
Just so you know, there is a neat way in ES6 to fill in arrays with sequential values, even make objects out of them: `Array.from(new Array(3),(val,index)=>({value: index}))`. This will give you `[{value: 0}, {value:1}, {value:2}]`. So for what you are trying to get you can replace the function with:
```suggestion
  getFamilyMembersCountArray = t => [
    ...[{ text: t('views.family.onlyPerson'), value: 1 }],
    ...Array.from(new Array(24), (val, index) => ({
      value: index + 2,
      text: `${index + 2}`
    })),
    ...[
      {
        text: t('views.family.preferNotToSay'),
        value: -1
      }
    ]
  ]
```
If that makes sense to you. How you pass the `t` also works, but I'd prefer to get it directly from props in the function instead of passing it during each render as seen bellow.

Reply:
We can't get t directly from the props, it needs to be dynamic so that translations change when language changed

Reply:
I see